package.json

```diff
@@ -40,9 +40,9 @@
     "eslint": "~2.2.0",
     "eslint-config-particle": "^1.0.1",
     "fs-extra": "^0.30.0",
-    "istanbul": "^0.3.22",
+    "istanbul": "^1.0.0-alpha.2",
```

Comment:
do we need to use the `alpha` pre-release version? if so, why? please share a github issue or similar if available

Reply:
Same as the comment below, when using istanbul (even with the latest 0.4.5 version) it fails with the `No coverage information` message error. This is also not working in the master branch. I saw the alpha version implementation here https://github.com/particle-iot/particle-api-js/blob/master/package.json#L61 and it worked.

Reply:
Additionally I will create several stories to remove istanbul in favor of nyc, as istanbul has been deprecated for a while.

Reply:
then the issue is down to our dependence on `istanbul` - take a crack at migrating to `nyc` here but time-box it to a couple of hours. be sure to capture your notes in a follow-on story if you aren't successful.

Reply:
Let's get rid of istanbul in the dependencies